pkgs/tools/misc/fdupes/default.nix

```diff
@@ -1,31 +1,28 @@
-{stdenv, fetchFromGitHub}:
+{ stdenv, fetchFromGitHub }:
 
 stdenv.mkDerivation rec {
-  name = "fdupes-20150902";
+  name = "fdupes-${version}";
+  version = "1.6.1";
 
   src = fetchFromGitHub {
-    owner = "jbruchon";
-    repo  = "fdupes-jody";
-    rev   = "414b1fd64c0a739d4c52228eb72487782855b939";
-    sha256 = "1q6jcj79pljm1f24fqgk8x53xz2x0p1986znw75iljxqyzbvw0ap";
+    owner = "adrianlopezroche";
+    repo  = "fdupes";
+    rev   = "v${version}";
+    sha256 = "19b6vqblddaw8ccw4sn0qsqzbswlhrz8ia6n4m3hymvcxn8skpz9";
   };
 
-  makeFlags = "PREFIX=\${out}";
+  makeFlags = "PREFIX=$(out)";
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "Identifies duplicate files residing within specified directories";
     longDescription = ''
-      FDUPES compares inodes' stats, hash sums, and byte by byte file
-      contents to find duplicate files within a set of directories and
-      then applies various actions to those sets, e.g.:
-      * remove some of the duplicates,
-      * turn all the files in a set into hardlinks.
+      <literal>fdupes</literal> searches the given path for duplicate files.
```

Comment:
Why `<literal>`?

Reply:
Because it's the command you would type in your shell

Reply:
Do you expect that some software will translate `<literal>` into something else, or that people will literally read `<literal>fdupes</literal>`?

Reply:
Not sure I understand your question...
It's in the long desc and I expected to be formatted like a command on https://nixos.org/nixos/packages.html
I'm pretty sure I've seen it done in another package longDesc but I may be wrong.

Reply:
Well, there is one package that uses `<literal>`, axolati from #30086, but there is no reason to expect that nixos website will replace it with formatting, and it is easy to check that `nix-env --description` prints it as is. These tags are only used in the documentation postprocessed with xsltproc, which certainly will not work on arbitrary longDescriptions.

Reply:
My mistake then. Will remove.

Reply:
No problem. When I first saw it in axoloti I though it was just a mistake, but when I saw you use it I suspected that you knew why it's useful.

Reply:
I guess I really wanted to believe that we could do some text formatting in the description!
Turns out I fooled myself... 😼